.github/ISSUE_TEMPLATE/word-definition.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Word definition
+about: Template to define undefined word
+title: "➕ Propose definition for [word]"
+labels: Type · Writing, Type · Definitions
+assignees: ''
+
+---
+
+title: definition
+slug: slug
+speech: 
+alt_words:
+  - alt word
+  - alt word
+  - alt word
+reading:
+  - text: 'Link Title'
+    href: https://google.com
```

Comment:
Do we want to use placeholder links that are not FAANG here?

Reply:
I think we can make them generic like https://www.link.com